### DISCUSSION TITLE
Not able to create "OpenAI Assistant", not even load existing OpenAI Assistant

### DISCUSSION THREAD

**Discussion by rajathash** [FlowiseAI/Flowise#2222] [Q&A]
Getting below error

Status: 500
Error: assistantsService.createAssistant - Error creating new assistant - retrievedAssistant.file_ids is not iterable

**Comment by Chatmindsolution** [comment]
I have the same problem! you are not alone, hope they will fix soon

**Comment by Chatmindsolution** [comment]
I get this too
Status: 500
Error: assistantsService.createAssistant - Error creating new assistant - 400 Unknown parameter: 'file_ids'
